autopush/db.py

```diff
@@ -247,7 +247,7 @@ def register_user(self, data):
                 for key, value in result["Attributes"].items():
                     try:
                         r[key] = self.table._dynamizer.decode(value)
-                    except AttributeError:
+                    except (TypeError, AttributeError):
```

Comment:
If value is not proper javascript, it throws a TypeError. If table or _dynamizer aren't initialized, they throw Attribute error. In either case, just take the value as specified and don't be clever.